src/runtime/database.cpp

```diff
@@ -396,6 +398,7 @@ std::string Database::open(bool wait, bool memory, bool tty) {
   const char *sql_tag_job = "insert into tags(job_id, uri, content) values(?, ?, ?)";
   const char *sql_get_tags = "select job_id, uri, content from tags where job_id=?";
   const char *sql_get_all_tags = "select job_id, uri, content from tags";
+  const char *sql_get_runs_table = "select run_id, time, cmdline from runs order by time ASC";
```

Comment:
I could have gotten sqlite to process the time to appropriate datetime format like SELECT strftime('%Y-%m-%d %H:%M:%S', time / 1000000000, 'unixepoch'), but decided to just default to the `Time` Library https://github.com/sifive/wake/blob/b4689381eb3bde982e8bad7502b1aca272d9c730/src/runtime/database.h#L55